### PR TITLE
Adding endpoint to edit comments by the comment owner and the project…

### PR DESCRIPTION
**Pull request by superalex**
… admins

### CODE REVIEW

File: taiga/projects/history/api.py [hunk 1]
```diff
@@ -57,6 +58,44 @@ def response_for_queryset(self, queryset):
...
+            return response.BadRequest({"error": _("deleted comments can't be edited")})
+
+        # comment_versions can be None if there are no historic versions of the comment
+        comment_versions = history_entry.comment_versions or []
+        comment_versions.append({
+            "date": history_entry.created_at,
+            "comment": history_entry.comment,
+            "comment_html": history_entry.comment_html,
+            "user": {
+                "pk": request.user.pk,
```
Comment (bameda) [inline taiga/projects/history/api.py]: Why are you using pk instead of id??

Comment (superalex) [inline taiga/projects/history/api.py]: It has been like that since the beginning...

File: taiga/projects/history/api.py [hunk 2]
```diff
@@ -56,6 +57,57 @@ def response_for_queryset(self, queryset):
 
         return response.Ok(serializer.data)
 
+    @detail_route(methods=['get'])
+    def comment_versions(self, request, pk):
+        obj = self.get_object()
+        comment_id = request.QUERY_PARAMS.get('id', None)
+        comment = services.get_history_queryset_by_model_instance(obj).filter(id=comment_id).first()
```
Comment (bameda) [inline taiga/projects/history/api.py]: You should use history_entry_id and history_entry instead of comment and comment_id because is very confuse

Comment (bameda) [inline taiga/projects/history/api.py]: ..and refactor `delete_comment` and `undelete_comment` too

**Comment by superalex** [comment]
@bameda I love when you review my PRs :+1: 

Updated!

**Comment by bameda** [comment]
Closed, changes moved to https://github.com/taigaio/taiga-back/pull/724

**Pull request closed without merging**